modules/ols-creating-the-credentials-secret-using-cli.adoc

```diff
@@ -5,21 +5,19 @@
 [id="ols-creating-the-credentials-secret-using-cli_{context}"]
 = Creating the credentials secret using the command line
 
-Create a file that is associated with the secret key used to access the API of your LLM provider.
+Create a file that is associated with the secret key used to access the API of your LLM provider. You can use API tokens to authenticate your LLM provider. Additionally, {azure-official} supports authentication using Entra ID.
 
 .Prerequisites
 
-* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role. Alternatively, you are logged in to a user account that has permission to create a secret to store the Provider tokens.
```

Comment:
Same question re https://80611--ocpdocs-pr.netlify.app/openshift-lightspeed/latest/configure/ols-configuring-openshift-lightspeed.html#ols-creating-the-credentials-secret-using-web-console_ols-configuring-openshift-lightspeed as the other procedure - I don't think using CLI vs web console means we need different permissions, but might be worth checking / testing this.

Reply:
My prior comment applies here, too.

Reply:
Same response, I would check if you need to mention the permissions just without the console part, i.e. something like:

```
* You are logged in to the {ocp-product-title} cluster as a user with the `cluster-admin` role. Alternatively, you are logged in to a user account that has permission to create a secret to store the Provider tokens.
```

Reply:
done